src/app/core/store/collections/collections.effects.ts

```diff
@@ -142,6 +144,19 @@ export class CollectionsEffects {
     })
   );
 
+  @Effect()
+  public createSuccess$: Observable<Action> = this.actions$.pipe(
+    ofType<CollectionsAction.CreateSuccess>(CollectionsActionType.CREATE_SUCCESS),
+    withLatestFrom(this.store$.pipe(select(selectCollectionsDictionary))),
+    mergeMap(([, collections]) => {
```

Comment:
The proper way of accomplishing what you want using NgRx would be to replace `@Effect()` by `@Effect({dispatch: false})` and `mergeMap` by `tap` operator without any return value.

We already use it like this in `CommonEffects` for example.

Reply:
This is what I was looking for. However, when I tried, it does not work. 
```  @Effect({dispatch: false})
  public createSuccess$: Observable<Action> = this.actions$.pipe(
    ofType<CollectionsAction.CreateSuccess>(CollectionsActionType.CREATE_SUCCESS),
    withLatestFrom(this.store$.pipe(select(selectCollectionsDictionary))),
    tap(([, collections]) => {
```
```Error:(148, 10) TS2322: Type 'Observable<Action | [CreateSuccess, Dictionary<Collection>]>' is not assignable to type 'Observable<Action>'.
  Type 'Action | [CreateSuccess, Dictionary<Collection>]' is not assignable to type 'Action'.
    Property 'type' is missing in type '[CreateSuccess, Dictionary<Collection>]' but required in type 'Action'.```

Reply:
Never mind, I figured that out.